### PR TITLE
Docs: update developer onboarding and others

### PR DESCRIPTION
**Pull request by ezdac**
This will update some aspects of the docs:
- Raspberry Pi installation, mentioned pip install
- Developer onboarding, updated permalinks, updated specs
- mention `RAIDEN_` envvar
- Remove Radar Relay mention
- [ ] **TODO** update changelog 3.0.0 release date once we release

### CODE REVIEW

File: docs/config_file.rst [hunk 1]
```diff
@@ -2,6 +2,9 @@ Raiden Config File
 ##################
 
 Raiden supports reading configuration parameters from a configuration file.
+Raiden configuration paramter can also be given as environment variables - the variable has to be prefixed
```
Comment (palango) [inline docs/config_file.rst]: s/paramter/parameter

**Review by palango — APPROVED** [review]
Looks good. Thanks for updating all those links! I didn't check all of those manually

File: docs/config_file.rst (hunk 1 above)
Comment (ezdac) [reply docs/config_file.rst]: resolved

**Comment by palango** [comment]
You can also merge this and update the date in another PR

**Pull request merged**